airbyte-commons/src/main/java/io/airbyte/commons/features/EnvVariableFeatureFlags.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.commons.features;
+
+public class EnvVariableFeatureFlags implements FeatureFlags {
+
+  @Override
+  public boolean usesNewScheduler() {
+    return System.getenv().containsKey("NEW_SCHEDULER");
```

Comment:
is this here and not in `Configs` because this is a feature flag and not configuration?

Reply:
there is already a `CONTAINER_ORCHESTRATOR_ENABLED` flag in `EnvConfigs`. should these be combined?

Reply:
can be done in a follow up PR

Reply:
Yeah that was the idea, putting it in its own interface since it is not really a config. I was thinking that those flags could move to a DB independently of the config. I made the interface with the scoped config that I suggest in the last brainstorm in mind.

Reply:
I can move `CONTAINER_ORCHESTRATOR_ENABLED` there after this is merged. I think there's some value in having them split for testing purposes.